lib/bambora/factories/response_adapter_factory.rb

```diff
@@ -6,7 +6,7 @@ module Bambora
   class ResponseAdapterFactory
     class << self
       def for(response)
-        content_type = response.headers['Content-Type']
+        content_type = response.headers['Content-Type'].split(';').first
```

Comment:
I don't know much about content types, do they all follow the same format, delimited by colons, with the first argument being the overarching name?

Reply:
Yeah, I forgot that they can include encoding data too. https://tools.ietf.org/html/rfc2616#section-7.2.1